content/applications/finance/accounting/getting_started/main_concept.rst

```diff
@@ -131,8 +123,22 @@ average price, LIFO (for countries allowing it) and FIFO.
 Easy retained earnings
```

Comment:
```suggestion
Retained earnings
```
I wouldn't use the word "easy" which is a bit too _marketingy_. You could just keep 'Retained earnings' to be consistent with other headings. Maybe "automatic", or "simplified" if you want to add an adjective? Not sure what's best here.